flexget/plugins/input/from_telegram.py

```diff
@@ -0,0 +1,219 @@
+                            'properties': {'username': {'type': 'string'}},
+                            'required': ['username'],
+                            'additionalProperties': False,
+                        },
+                        {
+                            'type': 'object',
+                            'properties': {
+                                'fullname': {
+                                    'type': 'object',
+                                    'properties': {
+                                        'first': {'type': 'string'},
+                                        'sur': {'type': 'string'},
+                                    },
+                                    'required': ['first', 'sur'],
+                                    'additionalProperties': False,
+                                }
+                            },
+                            'required': ['fullname'],
+                            'additionalProperties': False,
+                        },
+                        {
+                            'type': 'object',
+                            'properties': {'group': {'type': 'string'}},
+                            'required': ['group'],
+                            'additionalProperties': False,
+                        },
+                    ]
+                },
+            },
+
+            'only_new': {'type': 'boolean','default': True},
+            'entry': {
+                'type': 'object',
+                'properties': {
+                    'url': {'type': 'string', 'format': 'regex'},
+                    'title': {'type': 'string', 'format': 'regex'},
+                },
+                'additionalProperties': {'type': 'string', 'format': 'regex'},
+            },
+        },
+        'required': ['token'],
+        'additonalProperties': False,
+    }
+
+    @plugin.internet(logger)
+    def on_task_input(self, task, config):
+        #Load The Configs
+        token = config['token']
+        only_new = config['only_new']
+        entry_config = config.get('entry')
+        whitelist = config.get('whitelist',[])
+        types = config.get('types')
+
+        #Defaults
+        if not types:
+            types = ['private', 'group']
+
+        #Get Last Checked ID
+        update_id = task.simple_persistence.get(token+'_update_id')
+
+        #Get only new messages
+        params={}
+        if update_id and only_new:
+            update_id += 1
+            params['offset'] = update_id
+        
+        #The Target URL
+        url = "{domain}bot{token}/getUpdates".format(
+            domain=_TELEGRAM_API_URL,
+            token=token
+        )
+
+        #Get Telegram Updates
+        try:
+            response = task.requests.get(
+                url, timeout=60, raise_status=False, params=params
+            ).json()
+        except HTTPError as e:
+            raise plugin.PluginError("Error getting telegram update: {}" % e)
+            return
```

Comment:
no need to return after raise, it's unreachable

Reply:
Done!